core/modules/system/system.install

```diff
@@ -3632,6 +3635,19 @@ function system_update_1097() {
   }
 }
 
+/**
+ * Set the 'initial version' the current version, for pre-existing sites.
+ */
+function system_update_1098() {
+  $config = config('system.core');
+  // Set the initial version to anything lower than 1.30.0.
+  $initial_version = $config->get('initial_version');
+  if (!isset($initial_version)) {
+    $config->set('initial_version', '1.29.0');
```

Comment:
This is not "real" data, since this is not the actual initially installed version. Rather than setting this to a specific arbitrary version (anything lower than 1.30.0), should we perhaps set this to something like `0` or `0.0.0` or even just leave empty?

My concern is what we'll actually be sending to telemetry.